generate-test-data/README.md

```diff
@@ -105,14 +98,30 @@ This converter consumes and produces **CANONICAL** XER which is the same dialect
 and asn1jvm POJOs expect. It is characterized by octet strings with no spaces between the bytes,
 and compact XML with no line breaks or indentation.
 
-Usage example to convert a BSM MessageFrame from a file containing UPER-encoded binary to XER and
+Example. To convert the above SDSM XER to JER:
+
+```bash
+./converter-debug -p SensorDataSharingMessage -ixer -ojer sdsm1.xml > sdsm1.json
+```
+
+Another example to convert a BSM MessageFrame from a file containing UPER-encoded binary to XER and
 JER:
 
 ```bash
 ./converter-debug -p MessageFrame -iuper -oxer bsm_mf_noext_01.bin > bsm_mf.xml
 ./converter-debug -p MessageFrame -iuper -ojer bsm_mf_noext_01.bin > bsm_mf.json
 ```
 
+### converter-new-jer
+
+A converter generated from the `vlm_master` branch of the [mouse07410 fork](https://github.com/mouse07410/asn1c) of asn1c, which contains updates to the JER implementation that aren't present in the USDOT fork.  Specifically, it contains changes to the implementation of variable length bitstring encodings in JER.

```

Comment:
question: is this converter documentation the same as the debug version above? IE are parameters expected to be the same?

Reply:
Yes, added an example of using to that illustrates that.